src/cmap/auth/scram.ts

```diff
@@ -1,4 +1,4 @@
-import { saslprep } from '@mongodb-js/saslprep';
+import saslprep = require('@mongodb-js/saslprep');
```

Comment:
While this seems minimal this is not a desirable change, this syntax is for supporting legacy modules that declare their types as `export =` and it changes the javascript generated at runtime:


Before this change:
```ts
const saslprep_1 = require("@mongodb-js/saslprep");
// ...
(0, saslprep_1.saslprep)(password)
```

With this change:
```ts
const saslprep = require("@mongodb-js/saslprep");
// ...
saslprep(password)
```


IIRC using the saslprep's default (CJS default) export as a function breaks bundlers. 

Let's file a ticket to fix the typescript on the saslprep package and leave it on the version it is currently at for now.

Reply:
https://jira.mongodb.org/browse/NODE-5963

Reply:
just to confirm - we want to revert these changes too, right?

Reply:
Yep, re-request for viz, we do not want this change